app/assets/stylesheets/_settings.scss

```diff
@@ -1,895 +0,0 @@
-//  Foundation for Sites Settings
```

Comment:
you are effectively disabling Foundation here. Know that this is a pretty significant change to you project, that will be allowed, but cautioned.